src/frontend/packages/lib_video/src/components/common/PictureInPictureLayer/index.spec.tsx

```diff
@@ -15,7 +15,7 @@ describe('<PictureInPictureLayer />', () => {
   it('renders the main element only', () => {
     const MainContent = (
       <Box>
-        <Paragraph>main content</Paragraph>
+        <span>main content</span>
```

Comment:
```suggestion
        <p>main content</p>
```
I'd rather use `p` here, as it looks more like a real implementation (more semantic), even if it doesn't affect the purpose of the test.

Reply:
Depend if you have in mind a inline or a block element.